src/ntp64.rs

```diff
@@ -209,17 +245,33 @@ impl SubAssign<u64> for NTP64 {
 }
 
 impl fmt::Display for NTP64 {
+    /// By default formats the value as an unsigned integer in decimal format.  
+    /// If the alternate flag `{:#}` is used, formats the value with RFC3339 representation with nanoseconds precision.
+    ///
+    /// # Examples
+    /// ```
+    ///   use uhlc::NTP64;
+    ///
+    ///   let t = NTP64(7386690599959157260);
+    ///   println!("{t}");    // displays: 7386690599959157260
+    ///   println!("{t:#}");  // displays: 2024-07-01T15:32:06.860479000Z
+    /// ```
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        #[cfg(feature = "std")]
-        return write!(f, "{}", format_rfc3339_nanos(self.to_system_time()));
-        #[cfg(not(feature = "std"))]
-        return write!(f, "{:x}", self.0);
+        // if "{:#}" flag is specified, use RFC3339 representation
+        if f.alternate() {
+            #[cfg(feature = "std")]
+            return write!(f, "{}", format_rfc3339_nanos(self.to_system_time()));
```

Comment:
```suggestion
           write!(f, "{}", format_rfc3339_nanos(self.to_system_time()));
```

Reply:
That's not possible because of the following `#[cfg...]`.
Removing the `return` and the `;` (required because a `fmt::Result` must be returned) the compiler returns this error:
```log
error: expected `;`, found `#`
   --> src/ntp64.rs:263:73
    |
262 |             #[cfg(feature = "std")]
    |             ----------------------- only `;` terminated statements or tail expressions are allowed after this attribute
263 |             write!(f, "{}", format_rfc3339_nanos(self.to_system_time()))
    |                                                                         ^ expected `;` here
264 |             #[cfg(not(feature = "std"))]
    |             - unexpected token
    ```